packages/SwingSet/src/storeModule.js

```diff
@@ -1,4 +1,39 @@
-/* global VatData */
+/* global VatData globalThis */
+
+import { Far } from '@endo/marshal';
+
+import {
+  M,
+  makeScalarMapStore,
+  makeScalarWeakMapStore,
+  makeScalarSetStore,
+  makeScalarWeakSetStore,
+} from '@agoric/store';
+
+export {
+  M,
+  makeScalarMapStore,
+  makeScalarWeakMapStore,
+  makeScalarSetStore,
+  makeScalarWeakSetStore,
+};
+
+let MyVatData;
+if ('VatData' in globalThis) {
+  MyVatData = VatData;
+} else {
+  console.log('Emulating VatData');
```

Comment:
this makes me nervous, but it sounds like other people have a plan to rationalize access to `VatData`, so I'm not going to worry about it

Reply:
Thanks. Resolving.